picamera2/picamera2.py

```diff
@@ -992,8 +992,12 @@ def process_requests(self) -> None:
 
     def wait(self):
         """Wait for the event loop to finish an operation and signal us."""
-        result = self._future.result()
-        self._future = None
+        try:
+            result = self._future.result()
+        except Exception as e:
+            raise e
```

Comment:
Small question, does 'raise e' behave the same as 'raise' which is used lower down? (I assume it probably does?)